consensus/polybft/fsm.go

```diff
@@ -156,9 +156,11 @@ func (f *fsm) BuildProposal(currentRound uint64) ([]byte, error) {
 		}
 	}
 
-	if f.config.IsBridgeEnabled() {
-		if err := f.applyBridgeCommitmentTx(); err != nil {
-			return nil, err
+	if f.config.IsBridgeEnabled() && f.isEndOfSprint {
+		if f.isEndOfSprint {
+			if err := f.applyBridgeBatchTx(); err != nil {
+				return nil, err
+			}
 		}
```

Comment:
You have already checked `isEndOfSprint`.
```suggestion
        if err := f.applyBridgeBatchTx(); err != nil {
            return nil, err
        }
```